src/framework/extensions/internal/legacy/extpluginsloader.cpp

```diff
@@ -93,15 +93,21 @@ Manifest ExtPluginsLoader::parseManifest(const io::path_t& path) const
     io::FileInfo fi(path);
 
     Manifest m;
-    m.uri = Uri("muse://extensions/v1/" + fi.baseName().toLower().toStdString());
+    m.uri = Uri("muse://extensions/v1" + path.toQString().sliced(rootPath.size()).toLower().toStdString());
     m.type = Type::Macros;
     m.apiversion = 1;
     m.legacyPlugin = true;
 
-    auto dropQuotes = [](const String& str) {
+    auto dropQuotes = [](String str) {
+        while (str.back() == ';') {
+            str.truncate(str.size() - 1);
+        }
         if (str.size() < 3) {
             return String();
         }
+        if (str.startsWith(u"qsTr(\"") && str.endsWith(u"\")")) {
```

Comment:
strip off qsTr("  ") which is common in many plugin files (but really we should actually translate it...)